crates/gpui/Cargo.toml

```diff
@@ -133,7 +133,7 @@ pathfinder_geometry = "0.5"
 [target.'cfg(any(target_os = "linux", target_os = "freebsd"))'.dependencies]
 # Always used
 flume = "0.11"
-oo7 = { git = "https://github.com/zed-industries/oo7", branch = "avoid-crypto-panic" }
+oo7 = "0.4.0"
 
 # Used in both windowing options
 ashpd = { workspace = true, optional = true }
```

Comment:
I would bump this as well, because oo7 depends on it nowadays.

Reply:
Good catch, thanks!